modules/Users/GoogleApiKeySaverEntryPoint.php

```diff
@@ -0,0 +1,259 @@
+<?php
```

Comment:
This file having 43% coverage. It could be better. Please, add unit test to cover the rest of the new codes.

Reply:
@gymad 
There were some tests you wrote before the switch from codecept to phpunit. They're still accessible but were never moved over. I'm converting them now. I'll see what more I can build onto those.

Reply:
@gymad 
With my latest commit:
GoogleApiKeySaverEntryPoint  Lines:  75.90% ( 63/ 83)